apps/test/unit/lib/kits/maker/ui/SetupChecklistTest.js

```diff
@@ -48,7 +44,8 @@ describe('SetupChecklist', () => {
   });
 
   describe('test with expected console.error', () => {
-    allowConsoleErrors();
+    // Completely squelch console.error calls - ignore them and print nothing
+    beforeEach(() => console.error.reset());
```

Comment:
:stuck_out_tongue: Fine.  It's intentional cleanup of logspam in this test, but I can break it out.